src/background/messages/triggerPopup.ts

```diff
@@ -0,0 +1,17 @@
+import type { PlasmoMessaging } from "@plasmohq/messaging"
+
+import { eventPopupQueryString } from "~queries/eventPopupsQuery"
+import type { EventResponse } from "~types/eventTypes"
+import backgroundPopupCreate from "~utils/backgroundPopCreate"
+import { fetchStrapiContent } from "~utils/fetchStrapiContent"
+
+const handler: PlasmoMessaging.MessageHandler = async (req, res) => {
+  const { data } = await fetchStrapiContent<EventResponse>(
+    `api/events/${req.body.id}?${eventPopupQueryString}`
+  )
+
+  backgroundPopupCreate(data.pop_ups)
```

Comment:
can we add a test for this one?

I see two ways to test it:
- mock it and check that it's called once with an array of the length we expect, OR
- refactor it to create the loop here and just have the function called at each index to return height and width. That way narrow the focus of the util and we can actually test that it's called the number of times we expect and that it returns the exact value we expect at each. 

The second is much nicer if we need to mess with that big switch statement later on when we start creating new patterns for the popups and all that